server/datastore/mysql/hosts.go

```diff
@@ -472,6 +474,21 @@ LIMIT
 	return &host, nil
 }
 
+const hostMDMSelect = `,
```

Comment:
I'd appreciate input on whether this approach makes sense or if folks think we are better off mapping the data to these cases outside of the SQL layer.